src/main/java/org/altbeacon/beacon/service/scanner/CycledLeScanner.java

```diff
@@ -165,6 +165,14 @@ public void stop() {
         }
     }
 
+    public boolean getDistinctPacketsDetectedPerScan() {
+        return mDistinctPacketsDetectedPerScan;
+    }
+
+    public void setDistinctPacketsDetectedPerScan(boolean detected) {
+        mDistinctPacketsDetectedPerScan = detected;
+    }
```

Comment:
Can we drop these accessors?

This is an internal flag that's only needed for this class. Exposing it means long term support for this flag and limits the ability of this class to be refactored / modified without breaking API changes.